content/glossary/root-module.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Root Module"
+description: "Every Terraform configuration has at least one module, known as its root module, which consists of the resources defined in the .tf files in the main working directory. Root modules are the terraform configuration that we actually **apply**."
```

Comment:
```suggestion
description: "Every Terraform configuration has at least one module, known as its root module, which consists of the resources defined in the `.tf` files in the main working directory. Root modules are the terraform configuration that we actually **apply**."
```

Reply:
I've explained the difference between root modules and reusable child modules a bunch of times and I've never thought about mentioning that root modules actually manage state... duh. 👍